aip/general/0194.md

```diff
@@ -28,7 +28,8 @@ Clients **should not** automatically retry requests in which repeated runs
 would cause unintended state changes.
 
 **Note:** This AIP does not cover client streaming or bi-directional streaming.
-**Note:** Client side retry behavior for the client libraries is found in [AIP-4221](../client-libraries/4221.md).
+
+**Note:** Client side retry behavior for the client libraries is found in [AIP-4221](../client-libraries/4221).
```

Comment:
```suggestion
**Note:** Client side retry behavior for the client libraries is found in [AIP-4221](./client-libraries/4221).
```

I'm guessing the `..` should just be a `.`. How the directory structure vs. site structure lines up tricked me. Apologies.

Reply:
Are you 100% sure? I'm happy to trust you, but the way I got to my fix was just by deleting the .md from the URL that ended up on the page in #1177 - I worry changing it to `.` may accidentally introduce another error. Or it may work! I just don't know how to test

Reply:
You know...I don't think we need the link at all. If you look at https://google.aip.dev/128#further-reading, all of the AIP references are unlinked in source https://github.com/aip-dev/google.aip.dev/blob/18e067c6b7ae1d98a4bc32cee2a9722734853d92/aip/general/0128.md#L90-L97

I think the site has code to just auto-link AIP-**** texts.

Reply:
oh even better! I'll do a little refactor so it ends up in that `: see AIP-XXX` format :)